tests/foreman/ui_airgun/conftest.py

```diff
@@ -10,6 +12,33 @@
 
 LOGGER = logging.getLogger('robottelo')
 
+SESSION_TIMEOUT = 7000
```

Comment:
We do have max test duration defined in settings/robottelo/webdriver_desired_capabilities/maxDuration, why not reusing that constant? Imo that'd be better idea than making up new constant. And btw max test session duration on saucelabs is 90mins (5400secs), so there's no point in waiting longer than that.

Reply:
because the value here will be for the overall test run as it's created when entering the test